packages/ethereum-wallets/src/lib/index.ts

```
@@ -860,6 +870,30 @@
           logger.log("Wallet already connected");
         }
 
+        console.log(
+          "bannedNearAddressesPackage before",
+          bannedNearAddressesPackage
+        );
+        if (bannedNearAddressesPackage === null) {
+          console.log("fetching");
```

ESLint

[error] Unexpected console statement.

(no-console)